content/expressions/infix-ops.md

```diff
@@ -88,42 +88,45 @@ The full list of infix operators that are aliases for functions is:
 
 ---
 
-Operator   | Method         | Description
------------|----------------|------------
-`+`        | add()          | Addition
-`-`        | sub()          | Subtraction
-`*`        | mul()          | Multiplication
-`/`        | div()          | Division
-`%`        | rem()          | Remainder
-`<<`       | shl()          | Left bit shift
-`>>`       | shr()          | Right bit shift
-`and`      | op_and()       | And, both bitwise and logical
-`or`       | op_or()        | Or, both bitwise and logical
-`xor`      | op_xor()       | Xor, both bitwise and logical
-`==`       | eq()           | Equality
-`!=`       | ne()           | Non-equality
-`<`        | lt()           | Less than
-`<=`       | le()           | Less than or equal
-`>=`       | ge()           | Greater than or equal
-`>`        | gt()           | Greater than
-`>~`       | gt_unsafe()    | Unsafe greater than
-`+~`       | add_unsafe()   | Unsafe Addition
-`-~`       | sub_unsafe()   | Unsafe Subtraction
-`*~`       | mul_unsafe()   | Unsafe Multiplication
-`/~`       | div_unsafe()   | Unsafe Division
-`%~`       | rem_unsafe()   | Unsafe Remainder
-`<<~`      | shl_unsafe()   | Unsafe left bit shift
-`>>~`      | shr_unsafe()   | Unsafe right bit shift
-`==~`      | eq_unsafe()    | Unsafe equality
-`!=~`      | ne_unsafe()    | Unsafe non-equality
-`<~`       | lt_unsafe()    | Unsafe less than
-`<=~`      | le_unsafe()    | Unsafe less than or equal
-`>=~`      | ge_unsafe()    | Unsafe greater than or equal
-`+?`       | add_partial()? | Partial Addition
-`-?`       | sub_partial()? | Partial Subtraction
-`*?`       | mul_partial()? | Partial Multiplication
-`/?`       | div_partial()? | Partial Division
-`%?`       | rem_partial()? | Partial Remainder
+Operator   | Method         | Description                     | Note
+-----------|----------------|---------------------------------|---------------
+`+`        | add()          | Addition                        |
+`-`        | sub()          | Subtraction                     |
+`*`        | mul()          | Multiplication                  |
+`/`        | div()          | Division                        |
+`%`        | rem()          | Remainder                       |
+`%%`       | mod()          | Modulo                          | Since version `0.27.0`
```

Comment:
Two small things

First,

- I think this should be "Starting with version" as "Since" sounds like its been released.

Second,

We don't have a breaking change on record so the release this goes out in might be `0.26.1`

I think it's reasonable looking at the current open PRs to say `0.26.1` for now.